### DISCUSSION TITLE
Recipe titles on shopping list

### DISCUSSION THREAD

**Discussion by tjjunior** [mealie-recipes/mealie#902] [Feature Request]
_COMPLETE AND FILL THESE OUT_
- [x ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ x] Reviewed the existing feature requests and my feature has not been requested.
- [x ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
When holding a shopping list (either printed or on a phone), it can be helpful to know which recipes you're shopping for.  If they are out of lettuce, can I substitute cabbage?  Oh, it's just used as a garnish for that recipe so should be fine.

**(Optional): Suggest A Solution**
When showing a saved shopping list, include at the top the recipes it is pulling the ingredients from.
Not sure if it's currently being saved or just the actual ingredients, but ideally tie an array of the recipe ids/recipe names to the shopping list and display them when showing the list.  

**Additional Information**

- [x ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
This is available in the newest versions of Mealie :) The recipes are at the bottom, and toggleable on individual ingredients
